roles/ceph-common/tasks/installs/install_redhat_packages.yml

```diff
@@ -1,77 +1,84 @@
 ---
-- name: install red hat storage repository key
-  rpm_key:
-    key: "{{ ceph_rhcs_repository_path }}/RPM-GPG-KEY-redhat-release"
+- name: install redhat dependencies
+  package:
+    name: "{{ item }}"
     state: present
-  when:
-    - ceph_rhcs_iso_install
-
-- name: add red hat storage repository
-  template:
-    src: "{{ role_path }}/templates/redhat_storage_repo.j2"
-    dest: /etc/yum.repos.d/rh_storage.repo
-    owner: root
-    group: root
-    mode: 0644
-  when:
-    - ceph_rhcs_iso_install
+  with_items: "{{ redhat_package_dependencies }}"
+  when: ansible_distribution == "RedHat"
 
-- name: install dependencies
+- name: install centos dependencies
   package:
     name: "{{ item }}"
     state: present
-  with_items: "{{ redhat_package_dependencies }}"
+  with_items: "{{ centos_package_dependencies }}"
+  when: ansible_distribution == "CentOS"
 
-- name: install red hat storage ceph mon
+- name: install redhat ceph-mon package
   package:
     name: "ceph-mon"
     state: "{{ (upgrade_ceph_packages|bool) | ternary('latest','present') }}"
   when:
     - mon_group_name in group_names
 
-- name: install red hat storage ceph osd
+- name: install redhat ceph-osd package
   package:
     name: "ceph-osd"
     state: "{{ (upgrade_ceph_packages|bool) | ternary('latest','present') }}"
   when:
     - osd_group_name in group_names
 
-- name: install red hat storage ceph mds
+- name: install redhat ceph-mds package
   package:
     name: "ceph-mds"
     state: "{{ (upgrade_ceph_packages|bool) | ternary('latest','present') }}"
   when:
     - mds_group_name in group_names
 
-- name: install red hat storage ceph-common
+- name: install redhat ceph-fuse package
   package:
-    name: "ceph-common"
+    name: "ceph-fuse"
     state: "{{ (upgrade_ceph_packages|bool) | ternary('latest','present') }}"
   when:
     - client_group_name in group_names
 
-- name: install ceph-test
+- name: install redhat ceph-base package
   package:
-    name: ceph-test
+  package:
+    name: "ceph-base"
```

Comment:
I don't think Ansible should explicitly install the `ceph-base` package. It's a dependency that other packages (`ceph-mds`, `ceph-mon`, `ceph-selinux`) will automatically pull in.